core/src/edu/cornell/gdiac/main/LevelModel.java

```diff
@@ -657,20 +668,31 @@ public void makeTransparentObject(SimpleObstacle obstacle, AssetDirectory direct
 
 
   public void createFrames(SimpleObstacle obstacle, AssetDirectory directory, String key, int rows,
-      int cols, int frame) {
-    Animator.create(obstacle, directory, key, rows, cols, frame);
+      int cols, int frame, AnimationType type) {
+    Animator.create(obstacle, directory, key, rows, cols, frame, type);
   }
```

Comment:
You shouldn't need this method, this is the whole point of having the `Animator` utility class